### ISSUE TITLE
Reorganize locations of client and server code

### ISSUE THREAD

**Issue by bowlofeggs** [fedora-infra/bodhi#900]
Since the client and server code typically gets split into two different RPMs during packaging, it would be easier to separate them if they were more cleanly separated in our git repository. Something simple like moving them into bodhi/client/ and bodhi/server/ would make it a little easier to keep them separate in the spec file. Alternatively, we could create two separate Python packages for the client at server at the top level in this repository, or even start a new git repository for the bodhi client.

**Comment by bowlofeggs** [comment]
I've been working on updating the bodhi package in Rawhide, and I've hit a problem where the bodhi-client package depended on the bodhi-server package. I attempted to resolve this by creating a new python2-bodhi package to own the `bodhi/__init__.py` file, but unfortunately I then discovered that the init file contains server code. Due to this, I have decided to formally split the server and client code now so we have a clear separation between them.

Thus, this task is needed for #142.
